src/integrationTest/java/uk/gov/hmcts/darts/task/runner/impl/ProcessE2EArmRpoPendingAutomatedTaskIntTest.java

```diff
@@ -62,13 +57,10 @@ class ProcessE2EArmRpoPendingAutomatedTaskIntTest extends PostgresIntegrationBas
     private static final String ENTITLEMENT_ID = "SOME ENTITLEMENT ID";
     private static final String SOME_MASTER_INDEX_FIELD_ID = "SOME MASTER INDEX FIELD ID";
     private static final String SEARCH_ID = "SOME SEARCH ID";
+    private static final int AUTOMATION_USER_ID = -33;
```

Comment:
I was thinking about this but I think it couples test code verifications to production code values too much, although it maybe better to use the email address for readability and grab the ID from there but we get the same issue on config change it will break the tests.

Reply:
I guess they shouldn't change now that we have them set up, so probably little benefit making this much better.